gcp_iam_roles/dynatrace-gcp-monitor-metrics-role.yaml

```diff
@@ -1,8 +1,8 @@
-# For Kubernetes, Cloud Function counterpart is in setup.sh script
+# For Kubernetes. Cloud Function counterpart is in setup.sh script
```

Comment:
```suggestion
# For Kubernetes, the Cloud Function counterpart is in setup.sh script
```

Reply:
The . is deliberate. I can still add "The Cloud Function ...", though 😄